llvm/include/llvm/CGData/CodeGenData.h

```diff
@@ -164,6 +164,22 @@ publishOutlinedHashTree(std::unique_ptr<OutlinedHashTree> HashTree) {
   CodeGenData::getInstance().publishOutlinedHashTree(std::move(HashTree));
 }
 
+/// Initialize the two-codegen rounds.
```

Comment:
This comment isn't very useful (thanks to the function name 😄). Since the contents are also pretty simple, the comment probably isn't needed.